hedera-node/src/main/java/com/hedera/services/stream/RecordStreamFileWriter.java

```diff
@@ -66,8 +68,7 @@
 class RecordStreamFileWriter implements LinkedObjectStream<RecordStreamObject> {
     private static final Logger LOG = LogManager.getLogger(RecordStreamFileWriter.class);
 
-    private static final DigestType currentDigestType = DigestType.SHA_384;
-    private static final String COMPRESSION_EXTENSION = ".gz";
+    private static final DigestType currentDigestType = Cryptography.DEFAULT_DIGEST_TYPE;
```

Comment:
Is their an advantage of accepting platform `Cryptography`'s default digest type instead of explicitly using `DigestType.SHA_384`?

Reply:
The idea was to not have to go through all the classes and change them one by one _if_ at some point in the future the default digest type were changed. But now I'm wondering if the platform's `Cryptography` can vary independently of the services digests and whether the coupling is unwanted?

Reply:
That seems reasonable...it seems unlikely either Platform or Services will move away from SHA-384 any time soon!